sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/TextAnalyticsAsyncClient.java

```diff
@@ -237,11 +236,7 @@ public TextAnalyticsServiceVersion getServiceVersion() {
     @ServiceMethod(returns = ReturnType.SINGLE)
     public Mono<DocumentResultCollection<DetectLanguageResult>> detectBatchLanguage(
```

Comment:
One of the discussions with the adding `batch` to the methods was to add it at the end. so i.e. `detectLanguageBatch`
this is true for the other methods

Reply:
I will be doing that in an upcoming PR along with the change to add options parameter to List<String> method overload.

Reply:
Yes, It will fix in another API. Sameeksha and I splitting the workload now